src/SDKs/StreamAnalytics/StreamAnalytics.Tests/ScenarioTests/TransformationTests.ScenarioTests.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
```

Comment:
@shahabhijeet I believe the test infrastructure is set up similar to other test projects where we reference https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/AzSdk.test.reference.props which contains the Resource Manager NuGet.

Shouldn't that be updated if you want people to use the latest NuGet?